main.py

```diff
@@ -483,30 +476,28 @@ def main(cl_arguments):
                 phase="target_train",
             )
 
-            # Now that we've trained a model, revert to the normal checkpoint
-            # logic for this task.
+            # Now that we've trained the task specific module for the task,
+            # since we are accumulating the best parameters for
```

Comment:
I'm confused by this comment. I guess it doesn't apply in the finetune setting, but in the frozen setting, we're accumulating all of the best parameters in the checkpoint `model_state_target_train_best.th`, right? Where is that happening here? I only see us loading that checkpoint.

Reply:
Ohhh, is this comment saying we should load the best checkpoint from a particular task's target training?

Reply:
That makes sense.

Reply:
I had basically deleted target_train_best.th checkpoint since there is also model_state_%s_epoch_target_test.best_macro.th checkpoint. Is there a reason why the best parameters are  in both variables?